extensions/vscode/webviews/homeView/src/components/tree/TreeSection.vue

```diff
@@ -63,11 +59,22 @@ const toggleExpanded = () => {
     margin-left: auto;
   }
 
+  .tree-section-icon {
+    font-size: 13px;
+    margin-right: 4px;
+    display: initial;
+  }
+
   &.expanded:hover .actions,
   &.expanded:focus-within .actions {
     display: initial;
   }
 
+  &.expanded .tree-section-icon {
+    display: none;
+    margin-left: auto;
+  }
```

Comment:
Personally since `expanded` is available to us I think we should not render the codicon rather than use CSS to not display it.